README.md

```diff
@@ -78,12 +146,153 @@ val records: DataFrame = sqlContext.redshiftFile(path, Seq("name", "age"))
 val records: DataFrame = sqlContext.redshiftFile(path, "name varchar(10) age integer")
 ```
 
-Migration Guide
----------------
+## Parameters
+
+The parameter map or <tt>OPTIONS</tt> provided in Spark SQL supports the following settings.
+
+<table>
+ <tr>
+    <th>Parameter</th>
+    <th>Required</th>
+    <th>Default</th>
+    <th>Notes</th>
+ </tr>
+
+ <tr>
+    <td><tt>redshifttable</tt></td>
+    <td>Yes</td>
+    <td>No default</td>
+    <td>The table to create or read from in Redshift</td>
```

Comment:
It does if you put the query in `()` (so that its a subquery in the from clause).  We can add this as another feature.